### PR TITLE
fix: Re-phrase the Polygon documentation

### PR DESCRIPTION
**Pull request by LautaroPetaccio**

### CODE REVIEW

File: content/player/blockchain-integration/transactions-in-polygon.md [hunk 1]
```diff
@@ -20,23 +20,31 @@ As stated in the the official Polygon website, [Polygon](https://polygon.technol
 MATIC is Polygon’s native token. It is a cryptocurrency used to cover gas fees in the Polygon network, among other use cases. MATIC is to Polygon as ETH is to Ethereum.
 You can buy MATIC in most cryptocurrency exchanges.
 
-## Polygon in Decentraland’s Marketplace
+## Polygon in Decentraland’s Marketplace and Builder
 
-By using the Polygon network, and thanks to Decentraland’s DAO, users can list, sell and buy wearables for free. But **transactions in Polygon are not free**. The Decentraland Foundation covers the cost of the transactions in Polygon, but there is a “free transaction limit”.
+By using the Polygon network, and thanks to Decentraland’s DAO, users can list, sell and buy wearables in the Marketplace or publish their collections in the Builder without paying for the transaction gas.
```
Comment (nachomazzara) [inline content/player/blockchain-integration/transactions-in-polygon.md]: Do you think that it is worth mentioning `meta-transactions`?

**Review by nachomazzara — APPROVED** [review]
Great Job!!!!

File: content/player/blockchain-integration/transactions-in-polygon.md (hunk 1 above)
Comment (LautaroPetaccio) [reply content/player/blockchain-integration/transactions-in-polygon.md]: I've changed it to include `using the meta-transactions services`, would you mind checking it out?

File: content/player/blockchain-integration/transactions-in-polygon.md [hunk 2]
```diff
@@ -20,23 +20,31 @@ As stated in the the official Polygon website, [Polygon](https://polygon.technol
...
 
-## What is the free transaction limit?
+**Transactions in Polygon are not free**. The Decentraland DAO covers the cost of the transactions in Polygon so that users can enjoy many costless transactions in the Marketplace.  
 
-Transactions on the Polygon network cost gas. The Decentraland DAO covers the cost up to a certain limit, so that users can enjoy many costless transactions in the Marketplace.
+In order to enjoy costless transactions there are three conditions that need to be met:
 
-## Transactions on the Polygon network cost gas.
+- You need to be connected to Ethereum Mainnet.
+- The item you are intending to buy need to have a price of 1 MANA or higher.
```
Comment (cazala) [inline content/player/blockchain-integration/transactions-in-polygon.md:32]:
```suggestion
- The item you are intending to buy needs to have a price of 1 MANA or higher.
```

**Pull request merged**